sdk/confidentialledger/Azure.Security.ConfidentialLedger/README.md

```diff
@@ -199,31 +239,104 @@ ledgerClient.PostLedgerEntry(
 
 firstPostResponse.Headers.TryGetValue(ConfidentialLedgerConstants.Headers.TransactionId, out string transactionId);
 
+// Wait for the entry to be committed
+status = "Pending";
+while (status == "Pending")
+{
+    statusResponse = ledgerClient.GetTransactionStatus(transactionId);
+    status = JsonDocument.Parse(statusResponse.Content)
+        .RootElement
+        .GetProperty("state")
+        .GetString();
+}
+
 // The ledger entry written at the transactionId in firstResponse is retrieved from the default sub-ledger.
 Response getResponse = ledgerClient.GetLedgerEntry(transactionId);
+
+// Try until the entry is available.
+loaded = false;
+element = default;
+contents = null;
+while (!loaded)
+{
+    loaded = JsonDocument.Parse(getResponse.Content)
+        .RootElement
+        .TryGetProperty("entry", out element);
+    if (loaded)
+    {
+        contents = element.GetProperty("contents").GetString();
+    }
+    else
+    {
+        getResponse = ledgerClient.GetLedgerEntry(transactionId, subLedgerId);
```

Comment:
DO we need some sleeping logic here?

Reply:
I was trying to avoid any sleeps. This will eventually be addressed via short LROs.